web/src/components/tasks/RaffleBingoPaginator.tsx

```diff
@@ -0,0 +1,193 @@
+import { Content } from "@/components/Content";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { CtaContainer } from "@/components/njwds-extended/cta/CtaContainer";
+import { SingleCtaLink } from "@/components/njwds-extended/cta/SingleCtaLink";
+import { HorizontalStepper } from "@/components/njwds-extended/HorizontalStepper";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { ActionBarLayout } from "@/components/njwds-layout/ActionBarLayout";
+import { TaskHeader } from "@/components/TaskHeader";
+import { TaskStatusChangeSnackbar } from "@/components/TaskStatusChangeSnackbar";
+import { fetchTaskByFilename } from "@/lib/async-content-fetchers/fetchTaskByFilename";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { StepperStep, Task } from "@/lib/types/types";
+import { LookupTaskAgencyById } from "@businessnjgovnavigator/shared";
+import { TaskProgress } from "@businessnjgovnavigator/shared/userData";
+import { ReactElement, ReactNode, useEffect, useState } from "react";
+
+interface Props {
+  task: Task;
+  CMS_ONLY_STEPINDEX?: number;
+}
+
+export const RaffleBingoPaginator = (props: Props): ReactElement => {
+  const [stepIndex, setStepIndex] = useState(props.CMS_ONLY_STEPINDEX ?? 0);
+  const [stepLabels, setStepLabels] = useState<StepperStep[]>([]);
+  const [step, setStep] = useState<Task>();
+  const { business, updateQueue } = useUserData();
+  const [successSnackbarIsOpen, setSuccessSnackbarIsOpen] = useState<boolean>(false);
+
+  const { Config } = useConfig();
+
+  const raffleBingoStepFiles = ["raffle-license-step-1", "raffle-license-step-2"];
+
+  const getRaffleBingoStep = async (stepIndex: number): Promise<Task> => {
+    const fileName = raffleBingoStepFiles[stepIndex];
+    const step = await fetchTaskByFilename(fileName);
+    return step;
+  };
+
+  const isNotLastStep = (stepIndex: number): boolean => {
+    return stepIndex !== raffleBingoStepFiles.length - 1;
+  };
+
+  const isNotFirstStep = (stepIndex: number): boolean => {
+    return stepIndex !== 0;
+  };
+
+  const isLastStep = (stepIndex: number): boolean => {
+    return stepIndex === raffleBingoStepFiles.length - 1;
+  };
```

Comment:
[pebble] You have 2 functions here that are virtually the same; `isLastStep` and `isNotLastStep`. The only difference between these is the logic operator. Instead of having 2 separate functions you could achieve the same outcome by just negating the return value of one of them.

EX: `!isLastStep` returns the same value as `isNotLastStep`

Additionally, when we're defining names we opt for something like `isLastStep` over `isNotLastStep`. Including "not" in the name is another logical operation the reader has to take into account. When we do want to actually check for a condition that is "not", it's easier to read `!isLastStep`.

For that same reason I'd also recommend modifying `isNotFirstStep`:
```typescript
  const isFirstStep = (stepIndex: number): boolean => {
    return stepIndex === 0;
  };
```

Reply:
Looking back at the comments previously left on this PR. Lex had called out renaming these functions. And I do agree with that initial rec. What I don't agree with is to use the word "not" in the actual name. I didn't catch that when I was looking at this before. It's only with fresh eyes that I'm seeing this.